core/mapper/exprmapper/util/util.go

```diff
@@ -0,0 +1,119 @@
+package util
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+)
+
+func ConvertToString(val interface{}) (string, error) {
```

Comment:
not a huge deal, but if you can use the core/data/coercion package rather than having duplicate coercion, that would be better...

Reply:
Thanks @mellistibco  It will fix in another linker PR later as we discussed.